pipeline/pipeline.go

```diff
@@ -89,15 +96,17 @@ type Pipeline struct {
 
 	metricsHolder *metricsHolder
 
-	// some debugging shit
+	// some debugging stuff
 	logger          *zap.SugaredLogger
 	eventLogEnabled bool
 	eventLog        []string
 	eventLogMu      *sync.Mutex
 	inSample        []byte
 	outSample       []byte
-	totalCommitted  atomic.Int64
-	totalSize       atomic.Int64
+	outputEvents    atomic.Int64
+	outputSize      atomic.Int64
+	inputEvents     atomic.Int64
+	inputSize       atomic.Int64
```

Comment:
Not big deal, but let's put `input` above `output`.

Reply:
done